test-sbt/jvm/src/test/scala/zio/test/sbt/ZTestFrameworkSpec.scala

```diff
@@ -85,18 +99,14 @@ object ZTestFrameworkSpec {
     loggers.map(_.messages) foreach (messages =>
       assertEquals(
         "logged messages",
-        messages.mkString.split("\n").dropRight(1).mkString("\n").withNoLineNumbers,
+        messages.drop(1).mkString("\n").withNoLineNumbers,
         List(
-          s"${reset("info: ")}${red("- multi-line test")}",
-          s"${reset("info: ")}  ${Console.BLUE}Hello,",
+          s"${reset("info: ")}  ${red("- multi-line test")}",
+          s"${reset("info: ")}    ${Console.BLUE}Hello,",
           s"${reset("info: ")}${blue("World!")} did not satisfy ${cyan("equalTo(Hello, World!)")}",
-          s"${reset("info: ")}  ${assertSourceLocation()}",
-          s"${reset("info: ")}"
+          s"${reset("info: ")}    ${assertSourceLocation()}",
+          s"""${reset("info: ")}${ConsoleRenderer.render(Summary(0, 1, 0, ""))}"""
         ).mkString("\n")
-//          .mkString("\n")
```

Comment:
Dead code?

Reply:
Do you mean this whole method? It's not being called at the moment because we switched from the SBT logger to the `Console`-based one, but I'm planning to restore this test once we've switched over to assertions on the `Console` output.